node/src/gateway.ts

```diff
@@ -156,6 +156,13 @@ export interface Gateway {
      */
     newSignedProposal(bytes: Uint8Array, signature: Uint8Array): Proposal;
 
+    /**
+     * Create a proposal from a serialized proposal.
+     * @param bytes - Serialized proposal.
+     * @returns A proposal.
```

Comment:
I wonder if we would be better to have a simple, consistent description for all of these `newType(bytes: Uint8Array)` methods. How about, "Recreate a _type_ from serialized data"?
```suggestion
     * Recreate a proposal from serialized data.
     * @param bytes - Serialized proposal.
     * @returns A proposal.
```